### ISSUE TITLE
Response Body is empty when sending down an empty JSON array

### ISSUE THREAD

**Issue by bvanderhoof** [facebook/flipper#1165]
## 🐛 Bug Report
In the Network tab, if you send down an empty array from the server, the Response Body show as blank.

## To Reproduce
Send down an empty array in Node.js/Express.

e.g.

```
res.status(200).send([])
```

Result in the Network Tab for "Response Body" is blank instead of an empty array.

## Environment
Mac OS Flipper, React Native project

**Comment by cekkaewnumchai** [comment]
Thank you for reporting this to us. Is it on Android or iOS? If it's is on Android, can you verify the version of Flipper in `gradle.properties`?

**Comment by bvanderhoof** [comment]
The issue happened when the request was made from iOS simulator and the
response was viewed in the Mac OS flipper app.

**Comment by cekkaewnumchai** [comment]
Oh I'll look into it.

**Comment by mweststrate** [comment]
IIRC this was fixed, let us know if it isn't.

**Comment by arthurgeron-work** [comment]
@mweststrate  I'm suffering with this issue, already tried upping the project's flipper + update my desktop app, nothing changed. I'm using RN 0.71.3 (new arch disabled), I wonder what might be wrong.. Do you think it's worth opening a new issue? 
![Captura de Tela 2023-03-20 às 10 38 59](https://user-images.githubusercontent.com/71399856/226356887-abae78b3-a8b9-4fc8-ae68-2a8a1738f86f.png)

**Comment by Ittechnolog** [comment]
same issue
mac os + ios simulator, flipper 0.182.0 (rn 0.71.3 (new arch disabled))
<img width="758" alt="Снимок экрана 2023-03-21 в 12 02 03" src="https://user-images.githubusercontent.com/119858633/226559683-0795ddf2-cdfe-42e9-af5e-91b67609d3c0.png">

mac os + android is OK

**Comment by jankoJelic** [comment]
macOS + simulator, same error

**Comment by dabarnard** [comment]
For anyone else coming across this issue, take a look [here](https://github.com/facebook/flipper/issues/4673). It might be related.

TLDR: Either downgrade flipper to 0.176.1 or add `"Accept-Encoding": "gzip"` to request headers as interim fix

**Comment by Ahmedhamed77** [comment]
why is this closed? still happening on `0.191.0`
